src/optimizer/acc_hash_join_optimizer.cpp

```diff
@@ -12,36 +12,83 @@ using namespace kuzu::planner;
 namespace kuzu {
 namespace optimizer {
 
-void ASPOptimizer::rewrite(planner::LogicalPlan* plan) {
+void AccHashJoinOptimizer::rewrite(planner::LogicalPlan* plan) {
     visitOperator(plan->getLastOperator().get());
 }
 
-void ASPOptimizer::visitOperator(planner::LogicalOperator* op) {
+void AccHashJoinOptimizer::visitOperator(planner::LogicalOperator* op) {
     // bottom up traversal
     for (auto i = 0u; i < op->getNumChildren(); ++i) {
         visitOperator(op->getChild(i).get());
     }
     visitOperatorSwitch(op);
 }
 
-void ASPOptimizer::visitHashJoin(planner::LogicalOperator* op) {
+void AccHashJoinOptimizer::visitHashJoin(planner::LogicalOperator* op) {
```

Comment:
Leave a comment here that we always prioritize left to right sip, compared to right to left one.